srml/council/src/voting.rs

```diff
@@ -168,11 +201,17 @@ impl<T: Trait> Module<T> {
 		<VetoedProposal<T>>::remove(proposal);
 	}
 
+	/// Perform and return the tally result of a specific proposal.
+	///
+	/// The returned tuple is: `(approves, rejects, abstainers)`.
 	fn take_tally(proposal_hash: &T::Hash) -> (u32, u32, u32) {
-		Self::generic_tally(proposal_hash, |w: &T::AccountId, p: &T::Hash| <CouncilVoteOf<T>>::take((*p, w.clone())))
+		let vote_of = |w: &T::AccountId, p: &T::Hash| <CouncilVoteOf<T>>::take((*p, w.clone()));
+		Self::generic_tally(proposal_hash, vote_of)
 	}
 
-	fn generic_tally<F: Fn(&T::AccountId, &T::Hash) -> Option<bool>>(proposal_hash: &T::Hash, vote_of: F) -> (u32, u32, u32) {
+	fn generic_tally<F>(proposal_hash: &T::Hash, vote_of: F) -> (u32, u32, u32) where
```

Comment:
as per previous comment, consider changing to `fn generic_tally<F>(proposal_hash: &T::Hash, vote_of: F) -> (YaysCount, NaysCount, AbstainsCount) where`.